.github/CODEOWNERS

```diff
@@ -45,6 +45,9 @@
 # The Desktop Guide:
 /docs/desktop/** @Thraka
```

Comment:
This can be deleted. Originally it was going to be `/docs/desktop/wpf` but we changed it to `/docs/desktop-wpf`

Reply:
Done, @Thraka.